src/main/java/twistedgate/immersiveposts/client/ClientProxy.java

```diff
@@ -14,16 +14,12 @@
 /**
  * @author TwistedGate
  */
-public class ClientProxy extends CommonProxy{
+public class ClientProxy extends CommonProxy {
```

Comment:
Highly dislike this, please revert/remove this space and all the other ones that exist everywhere here.

Keep the formatting.